Hyper/Controllers/Bots/Spawn/CarSpawnStrategy.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Character.Vehicles;
+using Common;
+using OpenTK.Mathematics;
+using Physics.Collisions;
+using Physics.TypingUtils;
+
+namespace Hyper.Controllers.Bots.Spawn;
+internal class CarSpawnStrategy : AbstractBotSpawnStrategy
+{
+    public CarSpawnStrategy(Scene scene, Settings settings) : base(scene, settings)
+    {
+    }
+
+    public override void Despawn()
+    {
+        throw new NotImplementedException();
```

Comment:
implement it. We need to dispawn when the chunk under a car dispawns

Reply:
Also don't we need 2 car spawn strategies. one for spherical and one for normal?

Reply:
I think that proper car spawning should be in another PR., right now it's just a wrapper for car creation we had in Scene.
There are some things to consider here, like should we save the positions of spawned cars, so that the players car re-spawns in the same place it was last parked at? NPCs probably shouldn't, but it may be expected for a car